logstash-core/lib/logstash/instrument/metric_type/counter.rb

```diff
@@ -25,14 +25,5 @@ def execute(action, value = 1)
     def value
       @counter.value
     end
-
-    def to_hash
```

Comment:
I can revisit this, but IIRC I had issue when mixing it with a `to_json` method.

I think `LogStash::Json.dump`  was calling to_hash before calling the `to_json` :(